evalml/pipelines/utils.py

```diff
@@ -192,13 +192,13 @@ def generate_pipeline_code(element):
     base_string = "\nclass {0}({1}):\n" \
                   "\tcomponent_graph = [\n\t\t{2}\n\t]\n" \
                   "{3}" \
-                  "\nparameters = {4}\n" \
+                  "\nparameters = json.loads(\"\"\"{4}\"\"\")\n" \
```

Comment:
Adding in `json.loads` to convert the JSON back to Python. This handles both JSON null to Python None and JSON "false"/"true" to Pythonic booleans.

Reply:
Cool! I think all of our current components default parameters are json serializable but maybe we should encode that in a test? Just to make sure future components have this convention.

Reply:
@freddyaboulton Do you mean as a test for all components?

Reply:
Filed a new [issue here](https://github.com/alteryx/evalml/issues/1526) to track adding these tests